### PR TITLE
Fix current_position_nonexec filter so it only filters out current exec positions, not past

### PR DESCRIPTION
**Pull request by rlskoeser**
this was preventing acting faculty director with past exec membership from showing up on the staff page — they were listed, but always listed as past because the filter logic was catching on their other positions

### CODE REVIEW

File: cdhweb/people/models.py [hunk 1]
```diff
@@ -178,7 +178,8 @@ def project_manager_years(self):
     def _current_position_query(self):
         # query to find a person with a current cdh position
         # person *has* a position and it has no end date or date after today
-        return models.Q(positions__isnull=False) & (
+        today = date.today()
+        return models.Q(positions__start_date__lte=today) & (
             models.Q(positions__end_date__isnull=True)
             | models.Q(positions__end_date__gte=date.today())
         )
```
Comment (thatbudakguy) [inline cdhweb/people/models.py:185]: does this get used elsewhere? seems like it'd affect other things too (and glad we fixed it!)

**Review by thatbudakguy — APPROVED** [review]
thanks for figuring this out! does this resolve #368?

File: cdhweb/people/models.py (hunk 1 above)
Comment (rlskoeser) [reply cdhweb/people/models.py:185]: I think it gets used in multiple places, which is why we had a method for it. Yeah, not sure how we missed that one (although it's an edge case for sure)

**Comment by rlskoeser** [comment]
> thanks for figuring this out! does this resolve #368?

No, it doesn't — I think it's a separate bug that we should make an issue for. For #368 we need to include faculty director and any interim faculty directors at the beginning of the exec committee member list, and I'm not sure how to do that with the current queries we have without breaking other things. (Maybe we don't make it part of the current query and just look for faculty directors separately from exec...)

**Comment by rlskoeser** [comment]
@thatbudakguy how do you feel about the test coverage? I don't think this method is tested explicitly anywhere

**Comment by thatbudakguy** [comment]
i noticed that too. i'm fine with merging; doesn't seem like something we need to fix right now — might get revisited or invalidated as part of #232 anyway.

**Pull request merged**